packages/server/scripts/database.sql

```diff
@@ -1,18 +1,19 @@
 CREATE TABLE users (
-  user_id varchar(255) UNIQUE PRIMARY KEY NOT NULL,
+  user_id varchar(255) PRIMARY KEY NOT NULL,
   joined_date TIMESTAMP,
   last_login TIMESTAMP,
   first_name varchar(50) NOT NULL,
   last_name varchar(50) NOT NULL,
   email varchar(255) UNIQUE NOT NULL,
-  avatar varchar(255) NOT NULL
+  avatar varchar(255) NOT NULL,
+  password varchar(255)
 );
 
 CREATE TABLE Guild (
   guild_id VARCHAR(255),
   name VARCHAR(100),
   handler VARCHAR(50),
-  description VARCHAR(255),
+  description TEXT,
```

Comment:
Why is the difference of using `TEXT` compared to `VARCHAR` here?
I think we should at least have a limit of the amount of characters they can have? Maybe limit it to 500 characters?

Reply:
From this StackOverflow article (https://stackoverflow.com/a/25301046), the memory of TEXT and VARCHAR will be the same since the expected amount of characters used will be beyond 256 characters. It would be nice to limit the amount of characters used but since the memory usage is the same, it's better in my opinion to account for the possibility of clubs going over the 500 character limit.